src/docs/010_manual/20_install.adoc

```diff
@@ -134,12 +148,11 @@ Run the following command to ask docToolchain to download the template of your c
 ----
 --
 
-== Build
+== Build: Creating Templates.
 
-By now, the docToolchain wrapper `dtcw` resides in your project folder and the arc42 template is installed.
-Now let's render it as HTML and PDF.
+By now, the docToolchain wrapper `dtcw` should be in your project folder along with the arc42 template.
 
-You should be able to execute the following commands:
+Let's now render it as HTML and PDF. Run the commands below to execute this:
```

Comment:
Execute what?

Reply:
I rephrased it to

Now Let's render it to HTML and PDF. To do so, run the commands below: